nodes/quantum-algorithms/portfolio-optimisation/locales/en-US/portfolio-optimisation.html

```diff
@@ -0,0 +1,53 @@
+<script type="text/html" data-help-name="portfolio-optimisation"> <!-- Change name -->
```

Comment:
If we intend to release this node with the latest version, it would be nice to have the corresponding documentation.